smoke-tests/build.gradle

```diff
@@ -10,8 +10,8 @@ dependencies {
   testImplementation("com.fasterxml.jackson.core:jackson-databind:2.11.2")
   testImplementation("com.google.protobuf:protobuf-java-util:3.12.4")
   testImplementation("com.squareup.okhttp3:okhttp:3.12.12")
-  testImplementation("io.opentelemetry:opentelemetry-proto:0.10.0")
-  testImplementation("io.opentelemetry:opentelemetry-api:0.10.0")
+  testImplementation("io.opentelemetry:opentelemetry-proto:0.15.0")
```

Comment:
I don't really know if it's important, but the `fake-backend` app still uses otel-proto 0.10.0: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/smoke-tests/fake-backend/build.gradle#L28

Reply:
This is a good catch.  Fortunately, it seems to work ok with the 0.15.0 on the testing side here.  🤷

Reply:
As long as it works 🤷